### DISCUSSION TITLE
Is there a way to create a custom field and or a way to add arrays of fields?

### DISCUSSION THREAD

**Discussion by oscadev** [pocketbase/pocketbase#864] [Q&A]
For example, is there a way to create a "person" field with its own fields of "name, age, address", and then create a collection called "families" where people can add multiple "person" entries using the person model? Similar to how contentful works where you can create a custom field (model) with its own set of fields, and then use that field as a field when creating a collection model. 

Its 2 separate questions (can you create a custom field like "person", and can you make a collection take as many "persons" as someone wants to enter?)

**Comment by oscadev** [comment]
I assume I can create a collection called "person", and then do a relation to a specific when I create a "person" to a specific "family", so I mean more so "can this be done while inside the families editor portal page?"

**Reply by ganigeorgiev** [reply]
No, at the moment this is not supported. You'll have to create 2 collections and link them via a single/milti-relation type field,  but in order to add new `person` entries to a `family` entry, you'll have to create the `person` entries upfront so that they can be linked.

In the future we may add support for "nested" collections, but for now this is out of the scope for v1.0.0.
